src/index.html

```diff
@@ -1,13 +1,709 @@
+    <link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css">
   </head>
-  <body>
-    <h1>Miami</h1>
+
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__content">
+          <nav class="header__top">
+            <a href="#" class="logo">
+              <img
+                class="logo__image"
+                src="./images/logo.svg"
+                alt="logo"
+              >
+            </a>
+
+            <a
+              href="tel:+1 212-535-7710"
+              class="icon icon--phone"
+            >
+              <div class="icon__contacts">
+                The Met Fifth Avenue <br>
+                +1 212-535-7710 <br>
+                The Met Cloisters <br>
+                +1 212-923-3700
+              </div>
+            </a>
+
+            <a href="#menu" class=" icon icon--menu"></a>
+          </nav>
+
+          <h1
+            class="header__title"
+            data-aos="zoom-in"
+            data-aos-duration="2000"
+          >
+            Welcome
+            <br>
+            to The MET
+          </h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top">
+            <a href="#" class="logo">
+              <img
+                class="logo__image"
+                src="./images/logo.svg"
+                alt="logo"
+              >
+            </a>
+
+            <a href="#" class=" icon icon--cross"></a>
+          </div>
+
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link">
+                Home
+              </a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#about" class="menu__link">
+                About us
+              </a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#nowonview" class="menu__link">
+                Now on view
+              </a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#explore" class="menu__link">
+                Explore online
+              </a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#contacts" class="menu__link">
+                Contacts
+              </a>
+            </li>
+          </ul>
+
+          <a
+            href="tel:+1 212-535-7710"
+            class="menu__call menu__call--phone">
+            The Met Fifth Avenue<br>
+            +1 212-535-7710<br>
+          </a>
+
+          <a
+            href="tel:+1 212-923-3700"
+            class="menu__call menu__call--phone">
+            The Met Cloisters <br>
+            +1 212-923-3700
+          </a>
+
+          <a
+            href="tel:+1 212-535-7710"
+            class="menu__call menu__call--action">
+            Call to order
```

Comment:
According to the mockup, 'call to order' link should have underline on hover only, not by default.

![image](https://user-images.githubusercontent.com/78492113/218105396-281d6c54-7742-4ca2-880b-197be34add05.png)